mysite/ct/tests/integrate.py

```diff
@@ -7,12 +7,15 @@
 
 from django.test import TestCase
 from django.contrib.auth.models import User
-
+from mock import patch
```

Comment:
@alexbojko make this imports pep8 compliant
https://www.python.org/dev/peps/pep-0008/#imports